src/wallet/crypter.cpp

```diff
@@ -59,11 +59,9 @@ bool CCrypter::Encrypt(const CKeyingMaterial& vchPlaintext, std::vector<unsigned
     vchCiphertext = std::vector<unsigned char> (nCLen);
 
     EVP_CIPHER_CTX ctx;
-
-    bool fOk = true;
-
     EVP_CIPHER_CTX_init(&ctx);
-    if (fOk) fOk = EVP_EncryptInit_ex(&ctx, EVP_aes_256_cbc(), NULL, chKey, chIV) != 0;
+
+    bool fOk = EVP_EncryptInit_ex(&ctx, EVP_aes_256_cbc(), NULL, chKey, chIV) != 0;
```

Comment:
IMO this results in the same (remove code) as above.
The L66 EVP function only gets executed when fOk is true (which is always true at L66). fOk will then be set to true if the EVP function did not return 0.

I think the current code works right.